### DISCUSSION TITLE
Some helpful 3D prints

### DISCUSSION THREAD

**Discussion by mineshaftgap** [toverainc/willow#98] [Show and tell]
I have made two very crude 3D prints in order to achieve 2 things:
1. Block out the power LED glow without having to desolder it. For now I have it block the mute LED but not the button, if there is interest I can make another version that doesn't block the mute LED.
2. Stop ESP32-S3-Box from falling with an add on weighted base. I used the approach many have which is to have a place to insert pennies for weight, this right now is geared toward US pennies. If there is interest I can make other diameters if people provide them.

I made designed both to not need supports.

Since Github does not allow uploads of STL files I [zipped them up](https://github.com/toverainc/willow/files/11513051/stl-files.zip)

Here you can see what it looks like. There are a ton of ways it can be improved on, but it already makes it so much more pleasant:
![IMG_1169](https://github.com/toverainc/willow/assets/957262/df01f96e-e760-4c0b-ad6e-b135cc074475)
![IMG_1170](https://github.com/toverainc/willow/assets/957262/a26e9d35-ce46-40da-bdc6-926ca0509c05)

**Comment by kristiankielhofner** [comment]
Wow, Thanks!

Definitely adding this to the wiki!

For the record I never loved my suggestion of snapping the LEDs off either ;).

**Comment by mineshaftgap** [comment]
Excellent, feel free to add it to the repo if that is appropriate. Or I can make a PR if you would like.

**Comment by mineshaftgap** [comment]
Also totally understand if you don't want it in the repo since it is not Willow specific.

**Comment by ssurovich** [comment]
Great add-on - I was going to look at making something, but always great when someone else does the work before me!  

I think Im going to modify the original case a bit - I dont want the +5v out on the side like it is, going to see if there is any room to add some wires out of the back to make it look "cleaner".

**Comment by mineshaftgap** [comment]
@ssurovich I am not a fan of the USB C on the base with the angle it is at, I might look to see if there is a good adapter like one of these to help with the cable management: https://www.amazon.com/AreMe-Extender-Compatible-Notebook-Computers/dp/B0B2NJ3P3L

**Comment by kristiankielhofner** [comment]
Nor am I. Eventually we'd like to get some power supplies that make sense - correct angle on the connector, 1 AMP.

For the time being I caved and bought this (totally over-spec and expensive) cable just for the connector:

https://www.amazon.com/dp/B09FK2YXNQ?ref=ppx_yo2ov_dt_b_product_details&th=1

Ideally someone could find a much cheaper, more flexible USB 2.0 data rate + power USB-C to A equivalent, which I think would be perfect because I know we all have drawers full of 1 AMP USB A power adapters.

**Comment by kristiankielhofner** [comment]
Sorry, too excited to have mentioned this - if we want to move the cable and printing discussions to GH discussions that would be better as eventually we'll close this issue and it will effectively disappear.

**Comment by mineshaftgap** [comment]
@kristiankielhofner Absolutely let's move this to discussions, can an issue be migrated to a discussion on Github or should I make a new Discussion there?

**Comment by mineshaftgap** [comment]
I just checked, looks like an issue can be migrated to Discussions.

**Comment by blakadder** [comment]
Just slap these STLs on Printables or Thangs

**Comment by mineshaftgap** [comment]
Already have a draft I will make public on Printables soon. That does not help with version control so I will probably just make a repo.

**Comment by kquinsland** [comment]
> @kristiankielhofner Absolutely let's move this to discussions, can an issue be migrated to a discussion on Github or should I make a new Discussion there?

They [can be](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions)!

I too am not a fan of the USB-C port on the side of the S3 box modules. I've started thinking about how i'd "fix" that and it should be easy enough with a [dedicated USBC/PD negotiation module](https://www.aliexpress.us/item/3256804003562509.html?) and supply power through the pin headers on the bottom of the module.

**Comment by kristiankielhofner** [comment]
I don't use the side port on The ESP BOX at all, I'm pretty sure it's only there to support basic functionality when detached from the base. Or am I missing something here?

**Comment by mineshaftgap** [comment]
With the prints I did, I found that this right angle USB-C adapter works quite well
